src/main/java/cn/nukkit/command/CommandExecutor.java

```diff
@@ -38,12 +38,10 @@
      *                The sender of this command, this can be a player or a console.
      * @param command 要被发送的命令。<br>
      *                The command to send.
-     * @param label   这个命令的标签。<br>
-     *                Label of the command.
      * @param args    这个命令的参数列表。<br>
      *                Arguments of this command.
      * @return 这个命令执行是否执行成功。<br>whether this command is executed successfully.
      * @since Nukkit 1.0 | Nukkit API 1.0.0
      */
-    boolean onCommand(CommandSender sender, Command command, String label, String[] args);
+    boolean onCommand(CommandSender sender, Command command, String[] args);
```

Comment:
This change would cause binary compatibility issue with existing plugins, we could add a default modifier to the old one making it call this new version, but it would make plugins which uses this new version unnecessarily incompatible with NukkitX. I don't think it is worth it.